mne/preprocessing/xdawn.py

```diff
@@ -216,6 +216,11 @@ def __init__(self, n_components=2, signal_cov=None, correct_overlap='auto',
         """init xdawn."""
         self.n_components = n_components
         self.signal_cov = signal_cov
+        if reg == 'lws':
+            raise DeprecationWarning('`lws` has been deprecated for the `reg`'
+                                     ' argument. It will be removed in 0.11.'
+                                     ' Use `ledoit_wolf` instead.')
+            reg = 'ledoit_wolf'
```

Comment:
You do NOT need it as it was never released 


> On 29 juil. 2015, at 13:01, Romain Trachel <notifications@github.com> wrote:
> 
> In mne/preprocessing/xdawn.py:
> 
> > @@ -216,6 +216,11 @@ def __init__(self, n_components=2, signal_cov=None, correct_overlap='auto',
> >          """init xdawn."""
> >          self.n_components = n_components
> >          self.signal_cov = signal_cov
> > +        if reg == 'lws':
> > +            raise DeprecationWarning('`lws` has been deprecated for the `reg`'
> > +                                     ' argument. It will be removed in 0.11.'
> > +                                     ' Use `ledoit_wolf` instead.')
> > +            reg = 'ledoit_wolf'
> I did it here
> 
> —
> Reply to this email directly or view it on GitHub.
>